docs/02-running-gradle-plugin.md

```diff
@@ -0,0 +1,125 @@
+    │   │                   └── tests
+    │   │                       ├── AlfrescoIntegrationTest.java
+    │   │                       └── SampleIntegrationTest.java
+    │   └── resources
+    └── main
+        └── java
+            └── eu
+                └── xenit
+                    └── example
+                        └── repo
+                            └── DummyService.java
+
+```
+
+## Configuration
+
+To generate a correct DE bundle with your integration tests, we need the base package(s) that contain your integration test classes.
+
+You can configure this in the `alfrescoIntegrationTest.integrationTestPackages` property as an array.
+
+To be able to upload DE bundles to the Alfresco Repository, you must configure the `alfrescoDynamicExtensions.repository {}` block. This configuration is described in the [Dynamic Extensions Gradle Plugin documentation](https://github.com/xenit-eu/dynamic-extensions-for-alfresco/blob/master/documentation/Gradle_Plugin.md#configuration).
+
+The Repository configuration is also used to generate the correct remote URL for the test runner.
+
+### Example buildscript
+
+```groovy
+plugins {
+    id "eu.xenit.alfresco-remote-testrunner" version "2.0.0"
+    id "eu.xenit.de" version "2.0.1"
+}
+
+alfrescoIntegrationTest {
+    integrationTestPackages = ["eu.xenit.example.test.base.package"]
+}
+
+// Configuration of Dynamic Extensions is used to configure the repository
+alfrescoDynamicExtensions {
+    repository {
+        // Configure the URL to the Alfresco services endpoint
+        endpoint {
+            protocol = "http"
+            host = "localhost"
+            port = 8080
+            serviceUrl = "/alfresco/service"
+        }
+
+        // Configure admin credentials to install DE bundles
+        authentication {
+            username = "admin"
+            password = "admin"
+        }
+    }
+}
+```
+
+## Classpaths
+
+Contrary to "normal" java projects, where you have a compile and a runtime classpath, we will need to manage 3 classpaths here:
+
+* compile classpath
+* local runtime classpath: runtime classpath for the local JUnit stub.
+* remote runtime classpath: runtime classpath of the integration test bundle installed in Dynamic Extensions.
+
+Even though no user code is executed from the JUnit stub, the local runtime classpath needs to contain all classes that are imported in tests. The remote runtime classpath must contain all dependencies used by the integration tests, as all user code is executed there.
+
+Dependencies for the `integrationTest` sourceset are configured as follows:
+
+* `main` sourceset output is automatically included on the compile classpath and the local runtime classpath, not on the remote runtime classpath.
+* Additional dependencies in the `integrationTestImplementationRemote` are automatically embedded into the integration test DE bundle. They are on the compile, the local runtime classpath and the remote runtime classpath.
```

Comment:
(non-blocking) I suppose this is mean to say 'dependencies in the `integrationTestImplementationRemote` configuration'

Reply:
Yes, I did :smile: